src/fixtures/app.ts

```diff
@@ -0,0 +1,118 @@
+import cookieParser from "cookie-parser"
+import express, { Express } from "express"
+import _ from "lodash"
+
+import { errorHandler } from "@middleware/errorHandler"
+
+import GithubSessionData from "@classes/GithubSessionData"
+import UserSessionData from "@classes/UserSessionData"
+import UserWithSiteSessionData from "@classes/UserWithSiteSessionData"
+
+import { RequestHandler } from "@root/types"
+
+import {
+  mockUserSessionData,
+  mockUserWithSiteSessionData,
+  mockGithubSessionData,
+} from "./sessionData"
+
+/**
+ * @deprecated
+ */
+const attachSessionData: RequestHandler<
+  unknown,
+  unknown,
+  unknown,
+  unknown,
+  {
+    userSessionData: UserSessionData
+    userWithSiteSessionData: UserWithSiteSessionData
+    githubSessionData: GithubSessionData
+  }
+> = (req, res, next) => {
+  res.locals.userSessionData = mockUserSessionData
+  res.locals.userWithSiteSessionData = mockUserWithSiteSessionData
+  res.locals.githubSessionData = mockGithubSessionData
+  next()
+}
```

Comment:
Just a thought - I think it might still be useful to keep this around as a generic option for router tests that don't need to test different user profiles.

As discussed offline, we might consider making one with `SCREAMING_SNAKE_CASE` (to indicate that these are constants) for future use and deprecate this one

Reply:
Added in 960bd2e!